kits/peer-demo-showcase/apps/.env.example

```
@@ -0,0 +1,18 @@
+# Lamatic API Credentials & Endpoint
+LAMATIC_API_KEY=your_lamatic_api_key
+LAMATIC_PROJECT_ID=your_lamatic_project_id
+LAMATIC_API_URL=https://avadhutsorganization662-opopop597.lamatic.dev
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_API_URL key should go before the LAMATIC_PROJECT_ID key

(UnorderedKey)